src/core/tooltips/rfc.md

```diff
@@ -0,0 +1,87 @@
+## Tooltip Service
+
+This service allow us show tooltips in different places of our application.
+Underhood it uses `@konturio/ui-kit` tooltip component for show tooltip in theme style
```

Comment:
"to show" or "for showing"

Also i think all ui-kit components are style-compatible, no?

Reply:
> Also i think all ui-kit components are style-compatible, no?
Sure. Why you asking?

Reply:
yep